lbrynet/core/BlobManager.py

```diff
@@ -127,7 +140,10 @@ def create_tables(transaction):
                                 "    blob_hash text primary key, " +
                                 "    blob_length integer, " +
                                 "    last_verified_time real, " +
-                                "    next_announce_time real)")
+                                "    next_announce_time real, " +
+                                "    last_announce_time real, " +
```

Comment:
Do you have a specific use for `last_announce_time` in mind? If not, I'd prefer not creating it prematurely.